packages/superset-ui-chart-controls/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useTheme } from '@superset-ui/core';
+import { Tooltip as BaseTooltip } from 'antd';
+import { TooltipProps } from 'antd/lib/tooltip';
+
+export const Tooltip = (props: TooltipProps) => {
+  const theme = useTheme();
+  return (
+    <BaseTooltip
+      overlayStyle={{ fontSize: theme.typography.sizes.s, lineHeight: '1.6' }}
+      color={`${theme.colors.grayscale.dark2}e6`}
```

Comment:
Is the added semi-transparency required by design? Adding alpha channel this ways seems a little bit hacky. I could see this break if someone changed `theme.colors` to any non-hexcode format. But I don't know a good fix for this, so feel free to keep it (both adding a new color in `theme` or using `d3-color` for color calculation seem like an overkill).

Another suggestion, though, ideally these overrides (Superset default styles) should also allow run time overrides, which is not difficult to do:

```ts
export function Tooltip({
  overlayStyle,
  color,
  ...props
}: TooltipProps) {
  const theme = useTheme();
  const defaultColor = `${theme.colors.grayscale.dark2}e6`;
  return (
    <BaseTooltip
      overlayStyle={{ fontSize: theme.typography.sizes.s, lineHeight: '1.6', ...overlayStyle }}
      color={defaultColor || color}
      {...props}
    />
  );
};
```

Reply:
I added the overrides.
Alpha channel has already been used like that in some other components in main superset repo, including Tooltip (this Tooltip is copied from superset). I think it wasn't required by design, but I wanted to make it look similar to the old R-B tooltip.